### DISCUSSION TITLE
Is there a convenient "send all" feature?

### DISCUSSION THREAD

**Discussion by robinboening** [Python-Cardano/pycardano#123] [Q&A] [answered]
Hey there,

I've got two questions, but pretty related to each other:

1. is there a convenient way for sending all the ADA from a given address?
2. is there a convenient way for sending all native tokens from given address (using the lowest ADA amount)?

Thanks!

**Comment by astrojarred** [comment, marked as answer]
Hi Robin,

A convenient "send all" feature will be included in PR #89.

[See example here](https://github.com/astrojarred/pycardano/blob/add-wallet-classes/examples/wallet.py#L55)

If sending out all native tokens is something you could see being useful I will happily include it in the PR as well! Should be doable in just a couple lines of code with the framework that's already set up there.

**Reply by robinboening** [reply]
Hey @astrojarred thanks for the reply. I got inspired by your PR and put together what I needed to build a quick and simple send-all-assets feature as well. :)
